src/__test__/itemCount.test.js

```diff
@@ -0,0 +1,32 @@
+import { testShows } from '../modules/itemCount.js';
+
+global.fetch = jest.fn(() => Promise.resolve({
+  json: () => Promise.resolve(
+    [
+      {
+        show: {
+          id: 786,
+          url: 'https://www.tvmaze.com/shows/5/true-detective',
+          name: 'Cohle',
+          type: 'Scripted',
+          language: 'English',
+        },
+      },
+      {
+        show: {
+          id: 379,
+          url: 'https://www.tvmaze.com/shows/2/person-of-interest',
+          name: 'John',
+          type: 'Scripted',
+          language: 'English',
+        },
+      },
+    ],
+  ),
+}));
+describe('items counter tests using Jest', () => {
+  test('items count ', async () => {
+    const response = await testShows();
```

Comment:
- [ ] Kindly update the test for the count of all items based on the new implementation and please make sure that your test includes all edge cases like if there are 0 items on the page and use the `itemCounter` method that is used to count items instead of `testShows` method.